packages/common/src/types/NodeInfo.ts

```diff
@@ -0,0 +1,65 @@
+import { BakerId, Duration, Timestamp } from '../types';
+
+export type PeerId = string;
```

Comment:
```suggestion
export type PeerId = HexString;
```
Though it does seem to me the `PeerId` could just be a `HexString`. Like what is a `PeerId`?